### DISCUSSION TITLE
NaN issue comments

### DISCUSSION THREAD

**Discussion by jayvdb** [lowlighter/metrics#682] [General]
https://github.com/jayvdb/jayvdb/blob/7918d05/stats.svg shows "NaN issue comments".  I'm not sure when it started showing up, but I think it is quite recent, or I am quite unobservant.

**Comment by lowlighter** [comment]
GitHub was entirely down yesterday and came back with degraded API services, that may be what caused this

Maybe it'll autofix itself after a few days

**Reply by jayvdb** [reply]
It feels a bit odd that degraded API services would cause only one stat to have failed, and the rest are all fine.

It has fixed itself for my profile.

**Reply by lowlighter** [reply]
NaN can also occurs if the API timeout (ref #598). If you ever get this consistently it'll be (nearly) impossible to fix it because it's already a single query so it'd be more a GitHub issue. But at least you'll still get partial stats since it won't impact the others stats.

One a side note regarding degraded API services failing a single stat, it may be weird but there's a world where it could actually be possible.
On https://www.githubstatus.com issues and pull requests are their "own services" and I could see the API services actually forwarding or asking to these specific services. Could also explain why the reaction query fetching a single issue with GraphQL fails while fetching discussions works fine
